it_hardware/demo/website_view.xml

```diff
@@ -0,0 +1,430 @@
+                            <div class="container">
+                                <div class="row">
+                                    <div class="col-lg-2 offset-lg-5 o_colored_level">
+                                        <a href="/" class="o_footer_logo logo" data-bs-original-title="">
+                                            <img src="/web/image/it_hardware.ir_attachment_527" alt="" class="img-fluid mx-auto img o_we_custom_image" data-original-id="527" data-original-src="/web/image/it_hardware.ir_attachment_527" data-mimetype="image/webp" data-resize-width="690"
+                                            />
+                                        </a>
+                                    </div>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="bulk_order" model="ir.ui.view">
+        <field name="name">Bulk Order</field>
+        <field name="type">qweb</field>
+        <field name="key">website.bulk-order</field>
+        <field name="website_id" ref="website.default_website" />
+        <field name="arch" type="xml">
+            <t t-name="website.bulk-order">
+                <t t-call="website.layout">
+                    <div id="wrap" class="oe_structure oe_empty">
+                        <section class="s_website_form pt16 pb16 o_colored_level" data-vcss="001" data-snippet="s_website_form" data-name="Form" style="background-image: none;">
+                            <div class="container">
+                                <form action="/website/form/" method="post" enctype="multipart/form-data" class="o_mark_required" data-mark="*" data-pre-fill="true" data-success-mode="redirect" data-success-page="/contactus-thank-you" data-model_name="crm.lead">
+                                    <div class="s_website_form_rows row s_col_no_bgcolor">
+                                        <div data-visibility-condition="" data-visibility-between="" class="s_website_form_field mb-3 col-12 s_website_form_required col-lg-9" data-type="char" data-name="Field">
+                                            <div class="row s_col_no_resize s_col_no_bgcolor">
+                                                <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px;" for="olwptyu9ytwh">
+                                                    <span class="s_website_form_label_content">Name</span>
+                                                    <span class="s_website_form_mark"> *</span>
+                                                </label>
+                                                <div class="col-sm">
+                                                    <input type="text" class="form-control s_website_form_input" name="contact_name" required="1" placeholder="" id="olwptyu9ytwh" data-fill-with="name" />
```

Comment:
Try to give some meaningful id to the input and put it in for attribute of label in whole file
```suggestion
                                                <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px;" for="input-contact-name">
                                                    <span class="s_website_form_label_content">Name</span>
                                                    <span class="s_website_form_mark"> *</span>
                                                </label>
                                                <div class="col-sm">
                                                    <input type="text" class="form-control s_website_form_input" name="contact_name" required="1" placeholder="" id="input-contact-name" data-fill-with="name" />
```